### DISCUSSION TITLE
Is it possible to host PHP code on a different server and run it on my GitHub page?

### DISCUSSION THREAD

**Discussion by emullallphotography** [community/community#60720] [Programming Help]
### Body

If so, what are some free websites that will allow me to do this?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by AliAlmasi** [comment]
That's impossible, as your PHP files should be on your repo to run the GitHub pages.

**Comment by Misskittinn** [comment]
- [ ] @linknprk
